### PR TITLE
feat: add possibility to publish to custom-tag

### PR DESCRIPTION
**Pull request by leanmendoza**

### CODE REVIEW

**Comment by menduz** [comment]
looking good, would you update the readme before merge?

**Review by menduz — APPROVED** [review]

**Comment by leanmendoza** [comment]
> looking good, would you update the readme before merge?

yep, but before I'm testing it works as expected haha

File: oddish.ts [hunk 1]
```diff
@@ -424,7 +424,7 @@ const run = async () => {
 
     } else if (core.getInput("branch-to-next") === branch) {
       npmTag = "next";
-    } else if (customTag && customTag !== 'latest' && customTag !== 'next') {
+    } else if (customTag && customTag !== 'latest' && customTag !== 'next' && core.getInput("branch-to-custom-tag") === branch) {
```
Comment (menduz) [inline oddish.ts]:
```suggestion
    } else if (customTag && core.getInput("branch-to-custom-tag") === branch) {
```

This propbably should be the first condition of the if-chain?

**Comment by leanmendoza** [comment]
it works as expected :) 
https://github.com/decentraland/js-sdk-toolchain/actions/runs/6551127465/job/17791681898?pr=793

**Pull request merged**